setup.py

```diff
@@ -88,13 +88,27 @@ def get_ext_filename(self, name):
 
 
 def find_cx_Logging():
-    dirName = os.path.dirname(os.getcwd())
+    import subprocess
```

Comment:
I think I'd prefer this to be a warning that cx_Logging was not found. The documentation (and the warning itself) can include the commands required to set things up, but they shouldn't just be run automatically, I think.

Reply:
Only when compiling via sources will this happen, not in installations in general. For example, even if I had instructions, warning, or documentation I would prefer it to be automated and I think whoever opened the issue too. When I went to test issue #536, I had a good job to find out how to use cx_Logging, if it were automatic, it would be so simple.

Reply:
Ok. I can live with it. ;-)